### PR TITLE
Add changelog

### PR DESCRIPTION
**Pull request by ajsutton**
## PR Description
Adds a changelog with info about the upcoming (today hopefully) 0.11.2 release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+- In a future release the `--eth1-enabled` option will be removed. 
+  ETH1 will be enabled when an `--eth1-endpoint` is provided and otherwise disabled.
+
+## 0.11.2
+
+### Additions and Improvements
+
+- Updated to eth2 spec version v0.11.3.
+- Improved recovery from network changes. Peer's are now disconnected if they do not respond for a
```
Comment (macfarla) [inline CHANGELOG.md]:
```suggestion
- Improved recovery from network changes. Peers are now disconnected if they do not respond for a 
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+## Upcoming Breaking Changes
+
+- In a future release the `--eth1-enabled` option will be removed. 
+  ETH1 will be enabled when an `--eth1-endpoint` is provided and otherwise disabled.
+
+## 0.11.2
+
+### Additions and Improvements
+
+- Updated to eth2 spec version v0.11.3.
```
Comment (macfarla) [inline CHANGELOG.md]: nit: is it ETH2 or eth2

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+### Known Issues
+
+- This release provides support for the Witti testnet via `--network witti` however the configuration 
+  for this testnet is not yet stable and will likely differ from the one currently used.
+- The Schlesi testnet has been abandoned. The `--network schlesi` option will be removed in a future release.
+- Memory usage grows signficantly during periods of non-finalization.
+- Teku requires the ETH1 endpoint to keep historic world state available for at least the ETH1 voting period. 
+  This is typically more historic state than is kept when ETH1 nodes are pruning state. 
+  Work around is to connect to an archive node or configure the node to preserve a greater period of historic world state.
```
Comment (macfarla) [inline CHANGELOG.md]:
```suggestion
  Workaround is to connect to an archive node or configure the node to preserve a greater period of historic world state.  
```

**Review by macfarla — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 2 above)
Comment (ajsutton) [reply CHANGELOG.md]: Gave up and just called it "spec".  Technically it's the beacon-chain, fork-choice, honest-validator and p2p-interface specs.

**Review by rolfyone — APPROVED** [review]
LGTM

**Pull request merged**